adoc/Private-Registry-Administration.adoc

```diff
@@ -743,3 +743,206 @@ suse-registry-harbor-registry-65854df7bc-mrfnj     2/2     Running   0
 suse-registry-harbor-trivy-0                       1/1     Running   1          24h
 suse-registry-harbor-trivy-1                       1/1     Running   0          24h
 ----
+
+=== Backup and Restore
+
+The backup and restore for {spr} can be performed by link:https://documentation.suse.com/suse-caasp/4.2/html/caasp-admin/_backup_and_restore_with_velero.html[Velero]. Velero is an application that performs backup and restore, disaster recovery, and the migration of Kubernetes cluster resources and persistent volumes.
+
+When {spr} is configured to be deployed together with its internal services (database, Redis and filesystem storage for image and artifacts) Velero is able to fully cover the backup and restore for it. However, as Velero is only responsible for Kubernetes resources, when {spr} is deployed using external service providers, such as managed PostgreSQL database or object storage (Amazon S3/Azure Blob Storage), additional actions need to be performed to ensure the backup of data external to the Kubernetes cluster.
```

Comment:
```suggestion
When {spr} is configured to be deployed together with its internal services (database, Redis and filesystem storage for image and artifacts), Velero is able to fully cover the backup and restore for it. However, as Velero is only responsible for Kubernetes resources, when {spr} is deployed using external service providers, such as a managed PostgreSQL database or object storage (Amazon S3 or Azure Blob Storage), additional actions must be performed to ensure the backup of data that is external to the Kubernetes cluster.
```

Reply:
May I suggest keeping line length under 88 characters?

Reply:
As the whole document is not formatted under the 88 characters / line, I would prefer to keep it as it is.